storage/mdcb_storage.go

```diff
@@ -8,12 +8,19 @@ import (
 )
 
 type MdcbStorage struct {
-	local                 Handler
-	rpc                   Handler
-	logger                *logrus.Entry
-	CallbackonPullfromRPC *func(key string, val string) error
+	local                            Handler
+	rpc                              Handler
+	logger                           *logrus.Entry
+	CallbackOnPullCertificateFromRPC *func(key string, val string) error
 }
 
+const (
+	resourceOauthClient = "Oauth Client"
```

Comment:
I can see that `api key` is renamed to `ApiKey` (camelcase), do we need to do the same for Oauth client?

Reply:
done!